sdk/core/core-client/test/serviceClient.spec.ts

```diff
@@ -157,7 +157,7 @@ describe("ServiceClient", function() {
     await testSendOperationRequest(["1", "2", "3"], "Multi", false, "?q=1&q=2&q=3");
     await testSendOperationRequest(["1,2", "3,4", "5"], "Multi", false, "?q=1%2C2&q=3%2C4&q=5");
     await testSendOperationRequest(["1,2", "3,4", "5"], "Multi", true, "?q=1,2&q=3,4&q=5");
-    await testSendOperationRequest([], "Multi", true, "https://example.com/");
+    await testSendOperationRequest([], "Multi", true, "https://example.com");
```

Comment:
why is this change needed?

Reply:
I did a small optimization to not parse/stringify the url when no query params are added... but that means we don't canonicalize the url, so it doesn't get the trailing slash. We could add that functionality back in manually, but it didn't seem meaningful.

What do you think?

Reply:
Thanks for explaining! the current behavior looks good.